package.json

```diff
@@ -5,38 +5,38 @@
   "author": "Pedro Barco <pedro.barco@tecnico.ulisboa.pt>",
   "devDependencies": {
     "@bazel/ibazel": "^0.16.2",
-    "@graphql-eslint/eslint-plugin": "^3.10.7",
-    "@jest/types": "^29.0.1",
-    "@types/express": "^4.17.13",
-    "@types/jest": "^27.4.1",
-    "@types/node": "^17.0.25",
+    "@graphql-eslint/eslint-plugin": "^3.12.0",
+    "@jest/types": "^29.2.1",
+    "@types/express": "^4.17.14",
+    "@types/jest": "^29.2.0",
+    "@types/node": "^18.11.4",
     "@types/supertest": "^2.0.12",
-    "@typescript-eslint/eslint-plugin": "^5.20.0",
-    "@typescript-eslint/parser": "^5.20.0",
+    "@typescript-eslint/eslint-plugin": "^5.40.1",
+    "@typescript-eslint/parser": "^5.40.1",
     "@usrbinboat/eslint-config": "workspace:*",
     "@usrbinboat/prettier-config": "workspace:*",
     "c8": "^7.12.0",
-    "eslint": "^8.14.0",
+    "eslint": "^8.26.0",
     "eslint-config-prettier": "^8.5.0",
-    "eslint-import-resolver-typescript": "^2.7.1",
+    "eslint-import-resolver-typescript": "^3.5.2",
     "eslint-plugin-import": "^2.26.0",
-    "eslint-plugin-jest": "^26.1.5",
+    "eslint-plugin-jest": "^27.1.3",
     "eslint-plugin-jest-formatting": "^3.1.0",
-    "eslint-plugin-prettier": "^4.0.0",
-    "eslint-plugin-simple-import-sort": "^7.0.0",
-    "eslint-plugin-sonarjs": "^0.15.0",
-    "eslint-plugin-tsdoc": "^0.2.16",
-    "jest": "^27.5.1",
-    "jest-cli": "^28.0.1",
-    "prettier": "^2.6.2",
-    "supertest": "^6.2.3",
-    "ts-jest": "^27.1.4",
-    "ts-node": "^10.7.0",
+    "eslint-plugin-prettier": "^4.2.1",
+    "eslint-plugin-simple-import-sort": "^8.0.0",
+    "eslint-plugin-sonarjs": "^0.16.0",
+    "eslint-plugin-tsdoc": "^0.2.17",
+    "jest": "^29.2.1",
+    "jest-cli": "^29.2.1",
```

Comment:
Is this needed/used after migrating to `rules_jest`?

Reply:
`rules_jest` doesn't provide you with a runnable jest target so I'm using `jest-cli` to create one myself. This is only useful when you're trying to setup smoke/e2e tests via bazel.
I plan to add an example to this repo in the next couple of weeks.